src/qml/components/blockclockdial.cpp

```diff
@@ -15,7 +17,68 @@ BlockClockDial::BlockClockDial(QQuickItem *parent)
 , m_background_color{QColor("#2D2D2D")}
 , m_confirmation_colors{QList<QColor>{}}
 , m_time_tick_color{QColor("#000000")}
+, m_animation_timer{QTimer(this)}
 {
+    m_animation_timer.setTimerType(Qt::PreciseTimer);
+    m_animation_timer.setInterval(16);
+    m_delay_timer.setSingleShot(true);
+    m_delay_timer.setInterval(5000);
+    connect(&m_delay_timer, &QTimer::timeout,
+            this, [=]() { this->m_animation_timer.start(); });
+    connect(&m_animation_timer, &QTimer::timeout,
+            this, [=]() {
+                if (m_is_connected
+                    && getTargetAnimationAngle() - m_animating_max_angle < 1) {
+                    m_animation_timer.stop();
```

Comment:
The connecting animation seems to only be playing once and then won't play again on subsequent scenarios where we are trying to connect. I think because there's nothing starting this timer again?

Reply:
NVM not seeing this issue any more, am on plane wifi, maybe it was acting up somehow